src/ruleset/formats.ts

```diff
@@ -9,6 +9,10 @@ export class Formats extends Map<string, Format> {
     return new Formats([...this.entries()].filter(element => {return majorsToInclude.includes(element[0].split('.')[0]);}));
   }
 
+  excludeByMajorVersions(versionsToExclude: string[]): Formats {
```

Comment:
This is not needed anymore. My idea was to quickly disable the rule that fails for v3 and work on it later on. But we do have a solution (temporary?) in place now.

Reply:
If it's no longer needed, please remove it 😄

Reply:
Yup. Removed!